cmd/kogito/command/deploy/deploy_service.go

```diff
@@ -48,6 +49,7 @@ var (
 
 type deployFlags struct {
 	CommonFlags
```

Comment:
Is it possible to compose the `CommonFlags` with the `OperatorFlags`? So you wouldn't need to compose it here in the `deployFlags`, since we are repeating ourselves in lots of commands.

Reply:
done